.vscode/tasks.json

```diff
@@ -4,7 +4,7 @@
         {
             "label": "INSTALL (Upload to Device)",
             "type": "shell",
-            "command": "mvn install -f pom.xml `-Dmaven.javadoc.skip=true",
+            "command": "mvn install -f pom.xml -Dmaven.javadoc.skip=true",
```

Comment:
will remove

Reply:
I dropped these changes

Reply:
The current tasks are just setup for Powershell as the default shell. I use git bash as default and most of the commands are not compatible. Strangely if you set windows cmd as the default shell, the tick also cause problems. 
There is really no simple way to accommodate the two different default shells.

Reply:
Yeah, i struggled to find a good solution for this problem other than just adding another file and documenting that you would need to do some file renaming if you wanted to use a bash or cmd compatible file. I went with PowerShell as it would be the default for VS Code users.

Reply:
@TomKimsey ,
Good idea on the file renaming. 
I propose we change this file name to tasks_powershell.json, create a tasks_bash.json. Add `.vscode/tasks.json` to .gitignore and remove from tracking. Then

Reply:
I don't personally use VS Code development for Java, but I would think requiring developers to manually rename files should be a last resort. 

It's also not a clean procedure, because there is a sizable chance that the developer would open the project in VS Code and have to do the renaming there, which might require a restart of VS Code to pick up.

Why not just add different entries in a single `.vscode/tasks.json` like below? _It would make the development environment much more user-friendly during setup but achieve the same functionality._

```
{
    "version": "2.0.0",
    "tasks": [
        {
            "label": "INSTALL (Upload to Device) - Powershell",
            "type": "shell",
            "command": "mvn install -f pom.xml `-Dmaven.javadoc.skip=true",
            "group": "build"
        },
        {
            "label": "INSTALL (Upload to Device) - Bash",
            "type": "shell",
            "command": "mvn install -f pom.xml -Dmaven.javadoc.skip=true",
            "group": "build"
        },
        {
            "label": "PACKAGE (Create JAR)",
            "type": "shell",
            "command": "mvn package -f pom.xml",
            "group": "build"
        },
        {
            "label": "DEPLOY (Run on Device, No Debug) - Powershell",
            "type": "shell",
            "command": "mvn deploy -f pom.xml -P noDebug `-Dmaven.javadoc.skip=true",
            "group": "build"
        },
        {
            "label": "DEPLOY (Run on Device, No Debug) - Bash",
            "type": "shell",
            "command": "mvn deploy -f pom.xml -P noDebug -Dmaven.javadoc.skip=true",
            "group": "build"
        },
        ...
    ]
}
```

Reply:
This is clutter. Renaming a file is not a problem. Also every developer uses tasks differently. 
I am going to add a format task and a clean task that only removes target/classes. 
There is no reload required